systemFlake.nix

```
@@ -5,8 +5,8 @@
 , supportedSystems ? flake-utils-plus.lib.defaultSystems
 , inputs
 
-, channels ? { }
-, channelsConfig ? { }
+, streams ? { }
+, streamsConfig ? { }
```

```suggestion
, channels ? { }
, channelsConfig ? { }
, streams ? channels
, streamsConfig ? channelsConfig
```
For proper api deprecation protocol

---

Not resolved. `channels` and `channelsConfig` will not work the way its defined now